README.md

```diff
@@ -74,3 +74,21 @@ date: 2020-01-02 00:00:00
 ```
 
 http://yoursite.com/2020/01/02/foo/ ⇒ http://yoursite.com/2020/03/04/bar/
+
+### Redirect to a post
+
+If you specify a value without any slash, the article will be redirected to a post with that filename.
+
+For example, there is an existing post "source/_posts/foo-post.md" that is available at http://yoursite.com/2020/01/02/foo-post/,
+
+``` diff
+source/baz-page.md
+---
+title: baz
++ redirect: foo-post
+---
+```
+
+http://yoursite.com/baz-page ⇒ http://yoursite.com/2020/01/02/foo-post/
```

Comment:
🤔 

```diff
-http://yoursite.com/2020/01/02/foo-post/
+http://yoursite.com/foo-post/
```

Reply:
> For example, there is an existing post "source/_posts/foo-post.md" that is available at http://yoursite.com/2020/01/02/foo-post/

The page should redirect to post located at http://yoursite.com/2020/01/02/foo-post/
when a post couldn't be located (due to user's typo), then it will redirect to http://yoursite.com/foo-post/.